uint8[] clusterIDs
uint8[] clusters
ArmState[] rArm
ArmState[] lArm
duration[] timing
uint8 rRefFrame
uint8 lRefFrame
Object rRefFrameObject
Object lRefFrameObject